src/sentry/static/sentry/app/components/modals/sentryAppDetailsModal.jsx

```diff
@@ -23,6 +23,8 @@ export default class SentryAppDetailsModal extends React.Component {
   render() {
     const {sentryApp, closeModal, onInstall, isInstalled, organization} = this.props;
 
+    const overviewHtml = sentryApp.overview || '';
```

Comment:
This can probably just be called `overview`. It's technically not HTML yet.

Reply:
@mnoble Good suggestion. Fixed.